app/views/ads/new.html.erb

```diff
@@ -1,4 +1,3 @@
-<h1 class="title">Pomozimo žrtvama potresa</h1>
```

Comment:
I liked having this. It's also good for SEO.

Reply:
It was a duplicate of https://github.com/vlado/earthquake-croatia/blob/master/app/views/layouts/application.html.erb#L29